src/download.ts

```diff
@@ -64,17 +63,14 @@ export type GetFileContentResponse<T = unknown> = {
 /**
  * The response for a get metadata request.
  *
- * @property contentType - The type of the content.
  * @property metadata - The metadata in JSON format.
  * @property portalUrl - The URL of the portal.
  * @property skylink - 46-character skylink.
  */
 export type GetMetadataResponse = {
   metadata: Record<string, unknown>;
-  // TODO: Add back in once the endpoint supports these headers.
-  // contentType: string;
-  // portalUrl: string;
-  // skylink: string;
+  portalUrl: string;
+  skylink: string;
```

Comment:
I'm starting to confuse myself with Skylinks v2 -- but when called using an entryLink, does the endpoint return the metadata for the resolved dataLink? If so, should we return dataLink here since it might differ from the skylink used to make the request?

Is it getting too confusing when to expect a a `skylink` or `dataLink` field?

Reply:
I believe @ChrisSchinnerl said the skylink here would be the same one that was passed in. That's because in the front end we want to log which skylinks were accessed. In order to always return the dataLink we would need an additional header (which we may still add).